library/foundations/svgo.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  plugins: [
+    {
+      name: 'removeAttrs',
+      params: {
+        attrs: 'path:fill',
```

Comment:
This removes the `fill` attribute from any `path` elements during the `build-icons` script. Since the react icon props only apply to the parent `svg` element, any child `fill` attributes will not be overridden as expected.